mod/messages/views/default/forms/messages/send.php

```diff
@@ -20,6 +20,7 @@
 		'values' => $recipients,
 		'limit' => 1,
 		'required' => true,
+		'only_friends' => (bool) elgg_get_plugin_setting('friends_only', 'messages') ? true : false,
```

Comment:
when you cast it to a `bool` there is no need for the `? true : false` part any more ;)

Reply:
Correct. Fixed